code/ai/aicode.cpp

```diff
@@ -4376,7 +4376,7 @@ void ai_fly_to_target_position(vec3d* target_pos, bool* pl_done_p=NULL, bool* pl
 
 				if (wing_leader != Pl_objp) {
 					// not wing leader.. get my position relative to wing leader
-					get_absolute_wing_pos_autopilot(&goal_point, wing_leader, wing_index, aip->ai_flags[AI::AI_Flags::Formation_object]);
+					get_absolute_wing_pos(&goal_point, wing_leader, shipp->wingnum, wing_index, aip->ai_flags[AI::AI_Flags::Formation_object], true);
```

Comment:
Although `shipp->wingnum` is the same as `aip->wing`, use `aip->wing` as the argument to this function because it's consistent with the pattern throughout the rest of this block.